common/supplemental/supplementalData.xml

```diff
@@ -3143,7 +3143,8 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="gv" populationPercent="1.9" officialStatus="official"/>	<!--Manx-->
 		</territory>
 		<territory type="IN" gdp="9474000000000" literacyPercent="62.8" population="1296830000">	<!--India-->
-			<languagePopulation type="hi" populationPercent="41" officialStatus="official" references="R1066"/>	<!--Hindi-->
+            <languagePopulation type="hi" populationPercent="41" officialStatus="official" references="R1066"/> <!--Hindi-->
+            <languagePopulation type="hi_Latn" populationPercent="1"/> <!-- Hinglish -->
```

Comment:
Need to also update tools/java/org/unicode/cldr/util/data/country_language_population_raw.txt or this will get overwritten. Also:
- I am not sure that hi_Latn is the same as Hinglish.
- I believe that hi_Latn has usage greater than 1% in India - certainly on devices such as smartphones and computers - but I do not have data for this.